src/genotyper.cpp

```diff
@@ -651,19 +651,19 @@ vector<Genotyper::Site> Genotyper::find_sites_with_supbub(VG& graph) {
     // Unfold the graph
     // Copy the graph and unfold the copy. We need to hold this translation
     // table from new node ID to old node and relative orientation.
-    map<vg::id_t, pair<vg::id_t, bool>> unfold_translation;
+    unordered_map<vg::id_t, pair<vg::id_t, bool>> unfold_translation;
     auto transformed = graph.unfold(unfold_max_length, unfold_translation);
     
     // Fix up any doubly reversed edges
     transformed.flip_doubly_reversed_edges();
 
     // Now dagify the graph. We need to hold this translation table from new
     // node ID to old node and relative orientation.
-    map<vg::id_t, pair<vg::id_t, bool>> dag_translation;
+    unordered_map<vg::id_t, pair<vg::id_t, bool>> dag_translation;
     transformed = transformed.dagify(dagify_steps, dag_translation);
     
     // Compose the complete translation
-    map<vg::id_t, pair<vg::id_t, bool>> overall_translation = transformed.overlay_node_translations(dag_translation, unfold_translation);
+    unordered_map<vg::id_t, pair<vg::id_t, bool>> overall_translation = transformed.overlay_node_translations(dag_translation, unfold_translation);
```

Comment:
As part of these changes, do you want to adjust the signature of `map<id_t, vector<Mapping>> PathIndex::parse_translation(const Translation& translation)`? Or is that `Translation`-handling code logically distinct enough from this code that works with "translations" that get passed around in a different format?

And should we think about settling on a single internal format for translations? The `PathIndex` code that works with them uses `Mappings` so it can potentially express translations back to parts of original nodes, whereas this code here only thinks about entire nodes (because that's all it needs to). `Translation` objects can in general map from parts of nodes to parts of other nodes, but in practice they're always emitted following particular, not-documented-in-the-protobuf constraints that prevent that.